astropy/stats/tests/test_funcs.py

```diff
@@ -287,6 +286,39 @@ def test_bootstrap_multiple_outputs():
         assert bootresult.shape == (3, 2)
         assert_allclose(answer, bootresult, atol=1e-3)
 
+def test_jackknife():
```

Comment:
Since the jacknife() function depends on scipy, this test should be decorated with ``@pytest.mark.skipif('not HAS_SCIPY')`` (just as the test for bootstrap above). That will take care many of the travis fails you got.